docs/architecture/adr-013-non-interactive-default-rules-for-zero-padding.md

```diff
@@ -2,57 +2,69 @@
+In the naive approach if the block proposer aligned blobs one after another then there would be zero padding between blobs. Although this approach minimizes padding, it comes at the cost of large blob inclusion proofs because the hash of each share would be a subtree root of the proof. In a square with N shares, there would be N subtree roots. As a result, light nodes would need to download N subtree roots to verify blob inclusion proofs for this square which may be too large for resource constrained devices.
 
-Small blob sizes have the lowest ratio but also small inclusion proofs. Therefore increasing the proof size is not a problem until some threshold. It would increase the ratio of blob size to padding for small blobs which have the worst ratio.
+Small blob sizes have the lowest ratio of data to padding but also have small blob inclusion proofs. Since the size of blob inclusion proofs is an important constraint, we can establish a threshold to bound the number of subtree roots in a blob inclusion proof. From now on this threshold is called the `SubtreeRootThreshold` and it sets an upper bound for the number of subtree roots in a blob inclusion proof.
 
-Assuming there is a threshold, **the number of subtree roots in a proof**, where the proof size of a blob is acceptable, we can use this threshold to determine the index of the blob in the square. This would give us zero padding for blobs that are smaller than the threshold and non-zero padding for blobs that are larger than the threshold but still smaller than before.
+We can increase the `SubtreeRootThreshold` (and correspondingly the blob inclusion proof size) as long we are confident light nodes can process such a proof size. By increasing the subtree root threshold, we can place blobs closer together and therefore decrease the padding in the square. This is especially useful for small blobs since they have the lowest ratio of data to padding.
 
-Let's assume a good threshold assumption is that the number of subtree roots in a blob inclusion proof is acceptable if it is smaller than the `MaxSquareSize`. This would mean that the blobs smaller than the square size can use the index of one to get zero padding. Blobs that are larger than `MaxSquareSize` but smaller than `MaxSquareSize * 2` can use the index of two to get a maximum of 1 padding square. Blobs that are larger than `MaxSquareSize * 2` but smaller than `MaxSquareSize * 4` can use the index of 4 to get a maximum of 3 padding shares and so on.
+Given this threshold, the new non-interactive default rules would be:
 
-The new non-interactive default rules would be:
+- If the blob length (in number of shares) is smaller than `SubtreeRootThreshold` then the blob starts at an index that is a multiple of 1.
+- If the blob length (in number of shares) is larger than `SubtreeRootThreshold` then the blob starts at an index that is a multiple of the blob length divided by `SubtreeRootThreshold` rounded up.
```

Comment:
Should we add the fact that we always take min of ADR9 and ADR13 ?

Reply:
Resolved by https://github.com/celestiaorg/celestia-app/pull/1959/commits/4a4d33628a8d8c6f1f7664f34d18d7c4e9d77830